test/base/resources/kube.go

```diff
@@ -33,7 +33,15 @@ import (
 
 // EventSenderPod creates a Pod that sends a single event to the given address.
 func EventSenderPod(name string, sink string, event *CloudEvent) (*corev1.Pod, error) {
```

Comment:
If we add a new `add-tracing` arg, this function can be rewritten with `option` pattern by adding a `WithTracingAdded` option.

Reply:
Yes as above ^^